internal/pkg/aws/ec2/ec2.go

```diff
@@ -437,3 +438,25 @@ func (idx *routeTableIndex) IsPublicSubnet(subnetID string) bool {
 	}
 	return idx.mainTable.HasIGW()
 }
+
+// GetManagedPrefixListId returns the PrefixListId of a prefix list queried by name.
+func (c *EC2) GetManagedPrefixListId(prefixListName string) (*string, error) {
+	prefixListOutput, err := c.client.DescribeManagedPrefixLists(&ec2.DescribeManagedPrefixListsInput{
+		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String("prefix-list-name"),
+				Values: aws.StringSlice([]string{prefixListName}),
+			},
+		},
+	})
+
+	if err != nil {
+		return nil, fmt.Errorf("query returned error: %w", err)
```

Comment:
nit: we could provide more context wrapping the error:
```suggestion
		return nil, fmt.Errorf("describe managed prefix list with name %s : %w", prefixListName, err)
```